mods/storage/test/storage.unit.test.ts

```diff
@@ -0,0 +1,261 @@
+import Storage from "../src/storage";
+import chai from "chai";
+import sinon from "sinon";
+import sinonChai from "sinon-chai";
+import chaiAsPromised from "chai-as-promised";
+import fs, {Stats} from "fs";
+import {FonosService} from "@fonos/core";
+import {utils} from "./../src/utils";
+const expect = chai.expect;
+chai.use(sinonChai);
+chai.use(chaiAsPromised);
+const sandbox = sinon.createSandbox();
+import path from "path";
+import Fiber from "fibers";
+
+describe("@fonos/storage", () => {
+  let storage: Storage;
+
+  before(async () => {
+    storage = new Storage();
+    sandbox.stub(FonosService.prototype, "init").returns();
+  });
+  afterEach(() => sandbox.restore());
+
+  it("Should reject because is a directory", () => {
```

Comment:
The `it`  message must start in lowercase

Reply:
I agree with @Wardner , after all everything looks well.